notifications_utils/columns.py

```diff
@@ -121,7 +129,7 @@ def recipient_and_personalisation(self):
 class Cell:
     missing_field_error = "Missing"
 
-    def __init__(self, key=None, value=None, error_fn=None, placeholders=None):
+    def __init__(self, key=None, value=None, error_fn=None, placeholders=None, template_content_length=None):
         self.data = value
         self.error = error_fn(key, value) if error_fn else None
         self.ignore = Columns.make_key(key) not in (placeholders or [])
```

Comment:
we are initializing template_content_lenght in Cell here, but not creating a property for it? Where is this being used?